ra2ce/analysis/damages/damage_calculation/damage_network_base.py

```diff
@@ -168,8 +171,10 @@ def remove_unclassified_road_types_from_mask(self):
             )
             self._gdf_mask = df.loc[~(df["road_type"] == "none")]
 
-    ### Damage handlers
-    def calculate_damage_manual_functions(self, events, manual_damage_functions):
+    ### Damage handlers: TODO: move to dataclass ManualDamageFunctions
```

Comment:
Follow-up issue?

Reply:
Removed the TODO as it doesn't apply anymore.